### PR TITLE
Update the PHP transport classes section

### PR DESCRIPTION
**Pull request by HazAT**

### CODE REVIEW

File: src/collections/_documentation/platforms/php/index.md [hunk 1]
```diff
@@ -382,12 +422,19 @@ use Sentry\State\Hub;
...
+$transportFactory = new class implements TransportFactoryInterface {
+    public function create(\Sentry\Options $options): \Sentry\Transport\TransportInterface
+    {
+        return new SpoolTransport($spool);
+    }
+};
+
 $builder = ClientBuilder::create($options);
-$builder->setTransport($transport);
+$builder->setTransportFactory($transportFactory);
```
Comment (ste93cry) [inline src/collections/_documentation/platforms/php/index.md]: Somewhere after this call there should be a call to `$spool->flushQueue($httpTransport);`. This means that you should instantiate another transport (e.g. the HTTP transport) yourself and use it here

**Comment by ste93cry** [comment]
As a general thing I would suggest to always import all classes using `use` statements instead of having a mix of imports and FQCN

File: src/collections/_documentation/platforms/php/index.md (hunk 1 above)
Comment (HazAT) [reply src/collections/_documentation/platforms/php/index.md]: It's there, 3 lines below.

Comment (ste93cry) [reply src/collections/_documentation/platforms/php/index.md]: Ops, I didn't see it

**Pull request merged**